README.md

```diff
@@ -1 +1,240 @@
+-HTML
+-CSS
+-JS
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+The home page displays all the main information about the community as weel as speakers information
+The about page displays information about previous community work
+It has a mobile version too for each page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<a href="https://onlyunknown.github.io/Capstone-Mudole1/">Live view</a><br>
+<a href="https://www.loom.com/share/9c5d9f4d85cc48f3a8a7a3c053246570">The Recording<a>
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+It is a portfolio project so this could make the user able to see my recent work
+the frameworks that has been used and the program languages has been used.
+
+
+### Prerequisites
+
+Not necessary
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+ cd my-folder
+  git clone git@github.com:OnlyUnknown/Capstone-Mudole1.git
+
+<!--
+Example commands:
+
+```sh
+ 
+```
+--->
+
+### Install
+
+Install this project with:
+
+cd Capstone-Mudole1
+  gem install
+<!--
+Example command:
+
+```sh
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+-VScode 
+-The link of the live version above
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+
+👤 **Abdullah Al-Khureif**
+
+- GitHub: [@OnlyUnknown](https://github.com/OnlyUnknown)
+
+=======
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+Other Pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank <a href="https://www.behance.net/adagio07">Cindy Shin in Behance<a> for original design idea.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link to your license in your readme file is no longer active. Please correct it. It is critical to include a license in your project because it is a document that provides legally binding guidelines for use and imposes restrictions on how your project can be used. 📗 🙂